marquee/marquee.ino

```diff
@@ -341,7 +343,12 @@ void loop() {
     if (BitcoinCurrencyCode != "NONE" && BitcoinCurrencyCode != "") {
       msg += "    Bitcoin: " + bitcoinClient.getRate() + " " + bitcoinClient.getCode() + " ";
     }
-    scrollMessage(msg);
+    
```

Comment:
I have a small concern about doing this here.   The news feed will be advanced very minute, but it will only show a few of them.  Example, with a collection of 10 news feeds but it only displays every 5 minutes... you will see story 5 and 10 in the list.   We may need to restructure the section a little here.

Reply:
Thanks for checking the code.

Sorry, I disabled the news function and checked it.
I tried to fix it but how about you?
08b92e82576b447c2c324e7f012ca399890d554f

Since I'm using google translation, my sentences might be strange.

Reply:
Your translation looks very good.   Thanks for the contribution.  I just tested your branch here locally and it is looking good.   I am going to merge it into the 1.9 release, but I think I am going to add a few things to it -- like a max value and I may change the labels and the variable name.    
You did a great job in following the patter established on the project.  Thank you for the contribution.

Reply:
I have merged your changes in 1.9 -- I did make a few small changes.   Would you mind please testing out the 1.9 branch for me?   https://github.com/Qrome/marquee-scroller/tree/1.9
Thank you very much!

Reply:
I'm sorry ... The code check was too late.
I confirmed it with the master branch and there was no problem.
Thank you!

Reply:
Thanks for the contribution and testing!